### ISSUE TITLE
sample mapping & data processing for NOM for nmdc:sty-11-8ws97026 (PI Blanchard)

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#952]
**Deliverable this task is associated with**

_See Deliverables tab here: _

- [Add the Deliverable #]

**RACI**

_Tag people in their roles_
- Responsible: @corilo 
- Accountable: @bmeluch 
- Consulted: @mslarae13 @SamuelPurvine 
- Informed:

**Describe the the task**
- [x] The submission portal entry indicates 27 samples have NOM for nmdc:sty-11-8ws97026. Associated publication https://pubs.acs.org/doi/10.1021/acs.estlett.0c00748 
- [x] run NOM workflows

award DOI [10.46936/fics.proj.2016.49483/60006003](https://www.osti.gov/award-doi-service/biblio/10.46936/fics.proj.2016.49483/60006003)

**Criteria for completion**
- Mapping information of emsl data such that NOM workflows can be run.
- NOM results in mongo for study nmdc:sty-11-8ws97026

**Estimate people time**
- [Hours or days of people time. 1 person, 4 hours]

**Completion Date (Goal)**
- [GOAL!]

**Target Sprint Start & End Dates**
- Start:
- End:

**Tag Blocker/Contingent upon issues**
-

**Comment by SamuelPurvine** [comment]
Here are the mappings I have wrought 
[EMSL_49483_Blanchard_DataMapping_2024-11-27.xlsx](https://github.com/user-attachments/files/17940300/EMSL_49483_Blanchard_DataMapping_2024-11-27.xlsx)

**Comment by ssarrafan** [comment]
@bmeluch is this one still active?  I'll move to the next sprint since I think you're out this week. Happy holidays.

**Comment by bmeluch** [comment]
Yes still active, I've made a lot of progress on data processing thanks to Yuri, but not done yet

**Comment by aclum** [comment]
backlogging this until blocked issue (#1025) is resolved.

**Comment by bmeluch** [comment]
NOM raw data and results files have been uploaded to MinIO, JSON database files containing data_generation_set, data_object_set, workflow_execution_set have been submitted in the dev API.

**Comment by ssarrafan** [comment]
@bmeluch any update?

I'll move one more sprint to get this closed.  Thanks Bea

**Comment by bmeluch** [comment]
JSONs are now submitted so this is complete, thanks @ssarrafan !

**Comment by aclum** [comment]
These are not in prod b/c ingest happens once a week over the weekend and the records were added to prod on Monday 2/10.
